foreshadow/foreshadow.py

```diff
@@ -179,7 +179,7 @@ def fit(self, data_df, y_df):
             # Calculate parameter search space
             param_ranges = _param_mapping(deepcopy(self.pipeline), X_df, y_df)
 
-            self.opt_instance = self.optimizer(self.pipeline, param_ranges)
+            self.opt_instance = self.optimizer(self.pipeline, param_ranges, verbose=10)
```

Comment:
Since this is intended to setup a "generic" Optimizer class, does it make sense to pass a verbosity flag into it given that not all optimizers may not have an verbose constructor argument?

Reply:
I understand your concern. Do you have any suggestions? The goal here is to get some meaningful logs from the optimizer so that we can monitor the progress from the console while running the optimization.

Reply:
Lets defer this to an all-around upgrade of our logging policy throughout the framework. For the time being I would recommend removing the verbosity flag.

Reply:
Will remove the verbose argument for now.